pytorch_toolkit/tests/run_model_templates_tests.sh

```diff
@@ -1,11 +1,28 @@
 # This script should be run from the `pytorch_toolkit` folder
 # with one parameter -- work directory that will be used
 # for instantiating model templates and running tests
+WORKDIR=$(readlink -m $1)
 
+path_openvino_vars="${INTEL_OPENVINO_DIR:-/opt/intel/openvino}/bin/setupvars.sh"
```

Comment:
why do we need openvino here?

Reply:
OpenCV is required for mmcv. It is a must.